mappings/net/minecraft/util/dynamic/RegistryCodec.mapping

```diff
@@ -0,0 +1,28 @@
+CLASS net/minecraft/class_5380 net/minecraft/util/dynamic/RegistryCodec
+	COMMENT A codec for {@link SimpleRegistry}.
+	COMMENT
+	COMMENT <p>Compared to regular codec, this codec performs additional work when
+	COMMENT decoding, loading its elements from the given resource manager's JSON
+	COMMENT files.</p>
+	COMMENT
+	COMMENT @param <E> the registry's element type
+	COMMENT @see RegistryElementCodec
+	COMMENT @see RegistryOps
+	FIELD field_25504 delegate Lcom/mojang/serialization/Codec;
+	FIELD field_25505 registryRef Lnet/minecraft/class_5321;
+	FIELD field_25506 elementCodec Lcom/mojang/serialization/Codec;
+	METHOD <init> (Lnet/minecraft/class_5321;Lcom/mojang/serialization/Lifecycle;Lcom/mojang/serialization/Codec;)V
+		ARG 1 registryRef
```

Comment:
I assume this refers to the Registry's registry key?

Reply:
yes, and `registryRef` should be clearer than `registryKey`